source/SyntheticTopology/Overtness.lagda

```diff
@@ -0,0 +1,101 @@
+---
+title:        Overtness in Synthetic Topology
+author:       Martin Trucchi
+date-started: 2024-05-28
+---
+
+\begin{code}
+
+{-# OPTIONS --safe --without-K #-}
+
+open import MLTT.Spartan
+open import SyntheticTopology.SierpinskiObject 
+open import UF.Base
+open import UF.FunExt
+open import UF.PropTrunc
+open import UF.Sets
+open import UF.Sets-Properties
+open import UF.Subsingletons
+open import UF.SubtypeClassifier
+open import UF.UniverseEmbedding
+
+module SyntheticTopology.Overtness
+        (𝓤  𝓥 : Universe)
+        (fe : Fun-Ext)
+        (pe : Prop-Ext)
+        (pt : propositional-truncations-exist)
+        (𝕊 : Generalized-Sierpinski-Object fe pe pt 𝓤 𝓥)
+        where
+
+open import UF.ImageAndSurjection pt
+open import UF.Logic
+
+open AllCombinators pt fe
+open PropositionalTruncation pt hiding (_∨_)
+open Sierpinski-notations fe pe pt 𝕊
+
+
+\end{code}
+
+Overtness
+
+This notion is similar to compactness, in a dual meaning.
```

Comment:
Maybe say is the “dual notion of compactness”. I find “in a dual meaning” a bit unclear.

Reply:
Done!